### ISSUE TITLE
Call to function "file" failed: no file exists at scripts/init.sh.

### ISSUE THREAD

**Issue by yerzhant** [juliosueiras/terraform-lsp#69]
```
resource "openstack_compute_instance_v2" "master-1" {
...
  user_data       = file("./scripts/init.sh")
...
```
In VSCode there's an error:

`Call to function "file" failed: no file exists at scripts/init.sh.`

Though running `terraform apply` has no issues.

VSCode:
```
Version: 1.43.2
Commit: 0ba0ca52957102ca3527cf479571617f0de6ed50
Date: 2020-03-24T07:52:11.516Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 4.15.0-91-generic snap
```
Terraform extension 1.4.0

**Comment by juliosueiras** [comment]
this issue has been resolved in 0.0.10

**Comment by yerzhant** [comment]
Thanks.
